blog/_posts/2018-06-10-bsp.md

```diff
@@ -0,0 +1,214 @@
+minimal effort.
+
+As a result, end users are provided with the best developer experience and
+tooling developers can improve the quality of their integrations with less
+effort and time, without locking out a part of the Community from good Scala
+language support.
+
+### Relationship with the Language Server Protocol
+
+If you have heard of Microsoft's [Language Server Protocol][LSP], you may be
+wondering what's the relationship between these two protocols. BSP and LSP
+are complementary by design.
+
+While LSP specifies endpoints for communication between an *editor acting as
+a client* and a language server, BSP specifies endpoints between a *language
+server acting as a client* and a build server.
+
+It turns out that if we want to support Scala across editors like Visual
+Studio Code, `vim` or Sublime via a Scala language server, we also need to
+integrate with build tools to support features like compiler diagnostics or
+code navigation. One of the main focus of BSP is that it can be implemented
+alongside LSP.
+
+## The Build Server Protocol v1.0.0
+
+Today I announce version `v1.0.0` of the Build Server Protocol. This version
+has been the result of a fruitful collaboration with [Justin
+Kaeser][jastice], the [JetBrains] developer responsible for sbt support in
+[IntelliJ IDEA].
+
+Justin and I have spent the past months implementing a prototype of the Build
+Server Protocol between [IntelliJ IDEA] (client) and [Bloop] (server). The
+goal of this prototype was two-fold. On the one hand, we wanted to experience
+first-hand the challenges of implementing BSP and learn new insights to
+improve the protocol. On the other hand, we wanted to provide Scala
+developers a faster "Import Project" that would work across different build
+tools *now*, even though build tools don't yet provide implementations of
+build servers. Thanks to Bloop, users of [sbt], [mill] or [Maven] ---build
+tools that Bloop currently supports--- can try this prototype out.
+
+[BSP v1.0.0](https://github.com/scalacenter/bsp/blob/master/docs/bsp.md)
+has been designed to support the most basic language integration in an
+IDE/editor. IDEs can find all the modules defined in a workspace, ask for
+their dependencies, compile, test, run, and get notifications on modified
+build targets, among a few other features.
+
+Our IntelliJ-Bloop implementation doesn't yet implement all these endpoints.
+Instead, it focuses on providing a faster integration than the traditional
+project import. You can now import your project, browse Scala code with full
+language support and get compiler diagnostics from the build tool in
+IntelliJ's editor.
+
+How does this work? First, you export your project from your build to Bloop.
+This process takes usually ~10s for a medium-sized project (highly
+recommended to use [coursier] in your build for upmost speed) and exists so
+that Bloop can understand different build tools. Then, the IDE imports it
```

Comment:
What does `it` refers to in this sentence? I think it is "the bloop configuration" but you don't mention it.